frontend/src/styles/Search.css

```
@@ -0,0 +1,172 @@
+/* Search input/button & modal styles */
+
+.search-container {
+  display: flex;
+  align-items: center;
+  font-family: Lato;
+  font-weight: 400;
+}
+
+.search-input {
+  height: 21px;
+  padding-bottom: 5px;
+  padding-top: 4px;
+  padding-left: 4.95px;
+  font-size: 14px;
+  border: 0px;
+  border-radius: 3px 0px 0px 3px;
+  z-index: 101;
```

Input fields don't inherit the font from their parent, so you have to specify it for the input itself.

```suggestion
  z-index: 101;
  font-family: 'Lato', sans-serif;
```